subscrape/scrapers/moonbeam_scraper.py

```diff
@@ -136,18 +138,23 @@ def fetch_transactions(self, address, processor, reference=None):
             reference = address
         else:
             reference = reference.replace(" ", "_")
-        file_path = self.db_path + f"{reference}.json"
-        if os.path.exists(file_path):
-            self.logger.warning(f"{file_path} already exists. Skipping.")
-            return
-
-        self.logger.info(f"Fetching transactions for {reference} from {self.moonscan_api.endpoint}")
-        self.moonscan_api.fetch_and_process_transactions(address, processor)
 
-        payload = json.dumps(self.transactions[reference], indent=4, sort_keys=False)
-        file = io.open(file_path, "w")
-        file.write(payload)
-        file.close()
+        # Export the transactions to a JSON file
+        json_file_path = self.db_path + f"{reference}.json"
+        if os.path.exists(json_file_path):
+            self.logger.warning(f"{json_file_path} already exists. Skipping export.")
```

Comment:
if someone hasn't deleted their old output `.json` file, I vote to just delete and recreate the file instead of aborting and not saving anything. But I've left this warning and exit in since it's how you originally wrote it, @Tomen